outscale/data_source_outscale_image.go

```diff
@@ -197,30 +184,11 @@ func dataSourceOutscaleOAPIImage() *schema.Resource {
 func dataSourceOutscaleOAPIImageRead(d *schema.ResourceData, meta interface{}) error {
 	conn := meta.(*OutscaleClient).OSCAPI
 
-	filters, filtersOk := d.GetOk("filter")
-	executableUsers, executableUsersOk := d.GetOk("permission")
-	ai, aisOk := d.GetOk("account_id")
-	imageID, imageIDOk := d.GetOk("image_id")
-	if !executableUsersOk && !filtersOk && !aisOk && !imageIDOk {
-		return fmt.Errorf("One of executable_users, filters, or account_id must be assigned, or image_id must be provided")
+	req := oscgo.ReadImagesRequest{}
+	if filters, filtersOk := d.GetOk("filter"); filtersOk {
+		req.Filters = buildOutscaleOAPIDataSourceImagesFilters(filters.(*schema.Set))
 	}
```

Comment:
This is a singular data_source, therefore to avoid matching with multiple data_source user should add
```
filtersilters, filtersOk := d.GetOk("filter")
if !filtersOk {
   return fmt.Errorf("filters must be assigned")
}
```